test/registertask.js

```diff
@@ -133,6 +175,33 @@ describe('#registerTask()', function () {
       });
     });
 
+    describe('a new group with description', function () {
+      it('a single task', function () {
+        gf = 'grunt.registerTask(\'deploy\', \'desc\', [\'foo\'])';
```

Comment:
I'm not sure what you mean by this? In the previous code, the *gf* variable wasn't local when reaffected.